server.py

```diff
@@ -0,0 +1,59 @@
+#python -m uvicorn server:app --reload
+
+import sqlite_helpers
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import RedirectResponse, HTMLResponse
+from datetime import datetime
+
+from constants import HttpResponse, http_code_to_enum
+
+app = FastAPI()
+
+#maybe create the table if it doesnt already exist
+DATABASE_FILE = "urldatabase.db"
+sqlite_helpers.maybe_create_table(DATABASE_FILE)
+
+@app.post("/create_url")
+async def create_url(request: Request):
+
+    urljson = await request.json()
+    timestamp = datetime.now()
```

Comment:
we should be using the base `datetime` object, meaning
```suggestion
    timestamp = datetime.datetime.now()
```